### DISCUSSION TITLE
Please don't abandon this :)

### DISCUSSION THREAD

**Discussion by sgorange** [kingToolbox/WindTerm#1586] [General]
This is the best terminal available, and it would be a shame to abandon it :)

Keep up the good work !

**Comment by jonathansaha** [comment]
This ssh manager fills a very specific void. Most ssh managers with these features are paid or handicapped freemium, and I cannot use same software in both windows and linux. If you have a donation button on the website, I am sure many people will support you.

Please don't stop. You are doing God's work.

**Comment by Ducaz** [comment]
Pls don't abandon this project, the best Terminal so far, please keep it being the best.

**Comment by dcgsteve** [comment]
If it helps keep the project alive I would definitely donate! But not sure if it's money incentive or just lack of time or something else. Either way, I really hope you keep developing this - it's by far the *best* out of many unpaid and paid alternatives !!! 

Thank you for your work so far 👍
